doc/qos/SONiC_ROCEv2_support.md

```diff
@@ -0,0 +1,654 @@
+         |     |  +--rw oc-qos-ext:static-threshold           uint32
+         |     |  +--rw oc-qos-ext:dynamic-threshold          uint32
+         |     |  +--rw oc-qos-ext:pause-threshold?           uint32
+         |     |  +--rw oc-qos-ext:resume-threshold?          uint32
+         |     |  +--rw oc-qos-ext:resume-offset-threshold?   uint32
+         |     +--ro oc-qos-ext:state
+         |        +--ro oc-qos-ext:name?                      string
+         |        +--ro oc-qos-ext:pool                       -> ../../../../buffer-pools/buffer-pool/name
+         |        +--ro oc-qos-ext:type                       qos-buffer-type
+         |        +--ro oc-qos-ext:size                       uint32
+         |        +--ro oc-qos-ext:static-threshold           uint32
+         |        +--ro oc-qos-ext:dynamic-threshold          uint32
+         |        +--ro oc-qos-ext:pause-threshold?           uint32
+         |        +--ro oc-qos-ext:resume-threshold?          uint32
+         |        +--ro oc-qos-ext:resume-offset-threshold?   uint32
+         +--rw oc-qos-ext:buffer-priority-groups
+         |  +--rw oc-qos-ext:buffer-priority-group* [ifname priority-group]
+         |     +--rw oc-qos-ext:ifname            -> ../config/ifname
+         |     +--rw oc-qos-ext:priority-group    -> ../config/priority-group
+         |     +--rw oc-qos-ext:config
+         |     |  +--rw oc-qos-ext:ifname?           oc-if:base-interface-ref
+         |     |  +--rw oc-qos-ext:profile           -> ../../../../buffer-profiles/buffer-profile/name
+         |        +--ro oc-qos-ext:ifname?           oc-if:base-interface-ref
+         |        +--ro oc-qos-ext:priority-group?   string
+         |        +--ro oc-qos-ext:profile           -> ../../../../buffer-profiles/buffer-profile/name
+         +--rw oc-qos-ext:buffer-queues
+            +--rw oc-qos-ext:buffer-queue* [ifname queue]
+               +--rw oc-qos-ext:ifname    -> ../config/ifname
+               +--rw oc-qos-ext:queue     -> ../config/queue
+              +--rw oc-qos-ext:config
+              |  +--rw oc-qos-ext:ifname?    oc-if:base-interface-ref
+              |  +--rw oc-qos-ext:queue?     string
+              |  +--rw oc-qos-ext:profile    -> ../../../../buffer-profiles/buffer-profile/name
+              +--ro oc-qos-ext:state
+                 +--ro oc-qos-ext:ifname?    oc-if:base-interface-ref
+                 +--ro oc-qos-ext:queue?     string
+                 +--ro oc-qos-ext:profile    -> ../../../../buffer-profiles/buffer-profile/name
+
+```
+
+### 3.6.2 CLI
+
+
+#### 3.6.2.1 Configuration Commands
+All commands are executed in `configuration-view`:
+```
+sonic# configure terminal
+sonic(config)#
+```
+
+##### 3.6.2.1.1 Configure buffer pool
+Configure ingress and egress type buffer pools.
+```
+sonic(config)# qos buffer-pool <name> <shared-buffer-size-in-bytes> { [type ingress shared-headroom-size <xoff (included in the shared-buffer-size-in-bytes)>]  | [type egress ]}

```

Comment:
I believe the buffer size has different values for each chipset. Please provide how do identify and configure buffer-size for different hardware/chipset dynamically.

Reply:
We should document the buffer size for each platform in the user guide.